ddsc/core/ddsapi.py

```diff
@@ -16,6 +16,9 @@
 
 To cancel this operation, press Ctrl+C.
 """
+CONNECTION_RETRY_TIMES = 5
```

Comment:
I appreciate that these are constants at the top of the file and not buried as magic numbers, but I do think the number of retry constants has expanded enough to warrant a refactoring. As a follow up issue I'd suggest gathering these constants from the various files into a class where they could be grouped/structured/documented more easily.